sdk/python/lib/pulumi/resource.py

```diff
@@ -84,6 +84,13 @@ class ResourceOptions:
     An optional existing ID to load, rather than create.
     """
 
+    import_: Optional[str]
```

Comment:
You are correct with respect to the chosen name.

I had originally used `importId` in NodeJS and `import_id` here but changed to `import` and `import_` because I had received some early feedback that the first two choices didn't read well. I don't have terribly strong feelings here, so I am happy to roll with whatever the majority feels is best.

Reply:
I would be happy to keep `import` (the nice name) in NodeJS and `import_id` in Python.  These are going to differ anyway, and I think that `import_id` reads much better than `import_` FWIW.

Reply:
Using a trailing underscore to avoid conflicts with language keywords is PEP 8 style, FWIW: https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles (look for "`single_trailing_underscore_`: used by convention to avoid conflicts with Python keyword").